<launch>
    <arg name="mode"/>
    <arg name="width" default="0.08"/>
    <arg name="speed" default="0.1"/>
    <arg name="force" default="20.0"/>
    <arg name="epsilon_inner" default="0.002"/>
    <arg name="epsilon_outer" default="0.002"/>

    <node name="gripper_move" pkg="panda_controller" 
    type="gripper_move" respawn="false" output="screen">
        <param name="mode" value="$(arg mode)"/>
        <param name="width" value="$(arg width)"/>
        <param name="speed" value="$(arg speed)"/>
        <param name="force" value="$(arg force)"/>
        <param name="epsilon_inner" value="$(arg epsilon_inner)"/>
        <param name="epsilon_outer" value="$(arg epsilon_outer)"/>
    </node>
</launch>
